### DISCUSSION TITLE
How to write the format of Lang files.

### DISCUSSION THREAD

**Discussion by Robi-KenyaAA** [filamentphp/filament#8548] [Help] [answered]
How to write the format of lang files.
about Resource table Columns and form schema    ->translateLabel(),

example：Tables\Columns\TextColumn::make('community')->translateLabel() 

lang files ?????

thank you!!!

**Comment by leandrocfe** [comment, marked as answer]
[localization](https://laravel.com/docs/10.x/localization)

lang/en.json

```json
{
    "Community": "Comunidad"
}
```

**Reply by Robi-KenyaAA** [reply]
thank you !!!

---------
[localization](https://laravel.com/docs/10.x/localization)
- ->label(__('***Resource.community'))

lang/en/***Resource.php

```
return [
    'community' => 'Comunidad',
];
```
